### ISSUE TITLE
consider doing interviews - "getting to know us"

### ISSUE THREAD

**Issue by silky** [hahayes/hahayes-projects#12]
i was thinking that when you talk to people, you often find out about interesting things you didn't know.

then, it makes sense somehow to try and talk to a lot of people and find out about as many things as possible.

so then i thought of perhaps interviewing people that you see out and about on the street - maybe 2-3 a month - and then writing up the interviews say on tumblr or something. it could be called "getting to know us" and they idea would be just to learn about people. i'm not sure how to interview people, but what i want to know is:
- what things do you do?
- what do you want to do?
- favourite places? things done in those places?
- ???
- a few things you find intriguing? (totally based on the 'intriguing things' thing.)

stuff like that. i guess i'm interested in finding out what other people are interested in. we could record the interviews and they could go for maybe at most 15 minutes or something.

@mobeets i'm of course interested in your opinion here. i don't mean this to be similar to your existing intereview thing - i'd want a specific set of questions we ask; but maybe it flows a bit naturally, and i'd also want to try and keep it positive. i guess this is what the humans of melbourne thing does a bit.

similar things:
- http://www.meldmagazine.com.au/2013/06/humans-of-melbourne/
- http://www.humansofnewyork.com/

problems:
- how to pick people? biased on the way they look. is there a way to avoid that? (i.e. pick the person to your immediate left at exactly 6:54pm ...?? etc.)
- is it sufficiently different from humans of *? (i think so; because they don't post long interviews?)
- ???

@mobeets if you think it's a good idea let me know and i'll try and interview 2 people this month (even better if you agree to do it as well!!) and then we can write them up and start it off.

**Comment by silky** [comment]
(this - http://www.humansofnewyork.com/post/42601386010/is-this-for-fashion-week-nah-i-just-got-out - by the way is a glorious answer to any questions about outfits in the future.)

**Comment by mobeets** [comment]
i love this idea! i think randomly picking people is a great idea, like counting every ten people walking by or something, and doing that until you've found someone willing to be interviewed (i'm guessing that most people won't be willing, but we'll see!).

**Comment by mobeets** [comment]
something i think i learned from my interview thing is that for the most part people (okay, maybe my friends and not necessarily strangers, but still) want to fit the role but get nervous if that role is not clear. so i think what your initial speech is to these random people is really crucial.

i wonder in particular about the humansofnewyork guy--what's he say to these people when he approaches them? it's hard for people to be themselves right off the bat, i'd think. or at least you'd want to make it easy for the more shy people to be themselves as well, and not just the opinionated people.

**Comment by silky** [comment]
alright.

well over the next week or so let's decide on a good list of questions and format or whatever.

i.e. need to remember to take a photo; record the thing, get agreement to post the thing? (tell them what it's for?) etc?

and yeah. i agree - i don't know how to get people to tell you really "personal" things. probably help if they weren't in a group, and it was just a private chat. but if you're on the street, it might be hard. could be a coffee or something ...? (i.e. buy them a coffee and then do it? but still weird if it's in public?)

**Comment by mobeets** [comment]
also, i think the names of interview series are everything--so in line with your original goal of finding out interesting things you didn't know, i think maybe that's a good way of framing things. things like "is there something that a lot of your friends don't know about you?" or "is there something that you wish everyone knew?"

**Comment by silky** [comment]
nice; i like those.

**Comment by mobeets** [comment]
yeah, that's true! maybe a coffee is too intimidating unless you're charming as hell, which you might be, haha.

**Comment by silky** [comment]
hahaha! yeah i didn't consider that. i was thinking:

"hey, ah, i'm interviewing people for a thing, my name is noon, do you want to come and chat about stuff for a bit?"

haha. it'll be fun to see how horribly-wrong that goes.

**Comment by mobeets** [comment]
haha yeah, horribly wrong will be pretty fun though, in retrospect.

i really love this idea--definitely down to try to interview random people. i need to work now but maybe let's have a gchat some time to get more details sorted?

**Comment by silky** [comment]
sweet as!

let me know if you think of a better name and we can create a thing for it.

**Comment by silky** [comment]
[discussion moved to google docs for now; will close this issue once we've started the project.]

**Comment by silky** [comment]
david lynch has done a similar thing:
- http://www.openculture.com/2013/12/david-lynch-presents-the-interview-project.html
- http://interviewproject.davidlynch.com/www/#/all-episodes/121-spira

**Comment by mobeets** [comment]
wow, this looks awesome

**Comment by mobeets** [comment]
this is great, video interviews with people that seem really genuine and involving things that wouldn't be told otherwise. i mean, i'm sure a lot of people have done stuff like this, but somehow the fact that the whole premise is a focus on real people...doesn't that mean we can have as many "interview projects" as there are real people?

**Comment by silky** [comment]
oh yeah, i wasn't saying we should not do it - i'm definitely keen to give it a go!

there are some odd ones on here.
- http://interviewproject.davidlynch.com/www/#/all-episodes/041-tj

**Comment by silky** [comment]
could we call it "another interview project". or the standard programming naming strategy "yet another interview project (YAIP)".

maybe that would annoy david lynch. maybe it's funny that david lynch could possibly be annoyed with us. but i can't imagine that being the case. he'd surely approve.

watching a few of these; i really like the way it's done. i don't think we're set up to do recording in this way, though. but i think maybe at least saving the audio, and getting some photos is appropriate. i think videos would be great as well. but we need a team.

are you set up for that? i might be able to organise it. i'm not sure.

**Comment by mobeets** [comment]
yeah for sure, i am down to do it. i haven't watched any yet but i will soon.

i was just thinking last night actually, along the lines of how to randomly select people...maybe you could set it up in the opposite direction: instead of you choosing them, let them choose you. i was thinking about setting up a table somewhere (in street programming spirit) with a sign that says something like "TELL ME SOMETHING", and then just seeing what happens.

maybe i'd take their photo, or record their audio. haven't thought that far yet. i almost like the idea of it not being recorded at all, but having some sort of token for the interaction that both parties take away. like you swap photos or something. i don't know. not sure.

**Comment by silky** [comment]
hmm.

yeah. sitting at a table with a chair might make the interviews really easy.

i kind of like videos, or somehow recording it. because then other people not there can see what happens.

on the other hand, just kind of wanting to openly talk to people might be interesting. hmm

**Comment by mobeets** [comment]
just watched a ton of these interview project videos. they are so good! people are so different, it's amazing. this one's my favorite so far: http://interviewproject.davidlynch.com/www/#/all-episodes/013-barry
